src/config/dynamic-theme-fixes.config

```diff
@@ -676,6 +676,15 @@ nav.pagination {
 
 ================================
 
+blog.promods.net
+
+CSS
+body.custom-background {
+    background: none !important;
```

Comment:
The background-image is already dark by default, not sure why removing a non-bright image.

Reply:
The same as for the main website, this fix makes it *darker* if you dial the contrast to max.